lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -579,6 +579,26 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_global:
+  timeout:
+    config_get: "show running-config radius all"
+    config_get_token: '/^radius-server timeout (\d+).*$/'
+    config_set: '<state> radius-server timeout <timeout>'
```

Comment:
As the default values are outside of scope of netdev, i agreed that I would include them if they were provided to me. Feel free to provide the default values to me and I'll happily include these lines :)

Reason being is that it would significantly change the scope for our tickets, if we had to figure out what the default values were.

Thanks